cobrand_hackney/templates/happens_next.html

```diff
@@ -0,0 +1,62 @@
+<p>
+We now have a record on our system which will be assigned to an Officer. They
+will investigate this issue.
+</p>
+
+<p>
+If this issue is persistent or recurring, please read our
+<a href="https://hackney.gov.uk/noise">noise page</a>, which details what you
+can do and how we might be able to help you.
+</p>
+
+<p>
+We also recommend you record new instances against this report using this site.
+This will act as evidence if the case goes to court. If you don't add new
+reports within 3 weeks, we will assume that the nuisance has ceased and your
+case will be closed and no further action will be taken.
+</p>
+
+<p>
+Should the noise occur between 6.30pm and 02:00am on a Thursday, 9.00pm and
+02.00am on a Friday / Saturday, or 6.30pm and 02.00am on Sunday, please contact
+the out of hours team on <a href="tel:020 8356 4455">020 8356 4455</a>. If the
+noise is ongoing, we will visit you to try and witness the nuisance (other
+priorities permitting) and if necessary, take action to stop it.
+</p>
+
+<p>
+If the noise occurs at any other time please use this site and check the "I
+have reported this before" option at the start of the process. Out of hours
+monitoring visits can be arranged if additional reports show that the noise is
+regularly occurring outside our office hours. If, at any time we witness noise
+that amounts to a statutory nuisance, then a noise abatement notice would
+normally be served on the person responsible. If the noise continues after
+that, a prosecution would be considered, if we can prove that the notice has
+been breached. <b>Please note</b>: without witnessing the nuisance, there is
+only limited action we can take.
+</p>
+
+<p>
+If you have provided your contact details, we will keep you updated on
```

Comment:
There’s no way to get to this page _without_ having provided your contact details 🤔

Reply:
Seems to me that, in the case of an initial complaint about a residential property, this paragraph contradicts the paragraph that comes after it ("there will be no further action unless this issue is persistent and you report it again").

Reply:
> There’s no way to get to this page without having provided your contact details 🤔

Guess that statement is still accurate ;) Yeah, second bit is bit odd, guess I'll ask about it.